### PR TITLE
Carousel without animation

### PR DESCRIPTION
**Pull request by brockross**
Basic functionality of a carousel with a working temporary "next" button--animations and css refinements will come next

### CODE REVIEW

File: client/src/components/Carousel.jsx [hunk 1]
```diff
@@ -8,37 +8,67 @@ import CarouselItem from './CarouselItem.jsx';
...
-  justify-content: space-around;
+  margin: 0 0 20px 20px;
   width: 100%;
   height: 30%;
   overflow: hidden;
 `
 const Thumbnail = styled.div`
-  width: 15%;
-  margin: 1%;
+  flex: 1 0 100%;
```
Comment (bkwon94) [inline client/src/components/Carousel.jsx:17]: Is the flex property what you use to achieve the horizontal positioning of all the videos in the carousel? Did not know about flex at all so this is great to know.

File: client/src/components/Carousel.jsx [hunk 2]
```diff
@@ -8,37 +8,67 @@ import CarouselItem from './CarouselItem.jsx';
...
+  flex-basis: 20%;
+  margin-right: 20px;
   background: black;
+  order: ${(props) => props.order};
```
Comment (bkwon94) [inline client/src/components/Carousel.jsx:21]: Nice use of props in styled components. I haven't personally taken advantage of that.

File: client/src/components/Carousel.jsx [hunk 3]
```diff
@@ -8,37 +8,67 @@ import CarouselItem from './CarouselItem.jsx';
...
     super(props);
+
+    this.state = {
+      position: 0
+    }
+  }
+
+  getOrder(itemIndex) {
+    const { position } = this.state
+    const { videos } = this.props
```
Comment (bkwon94) [inline client/src/components/Carousel.jsx:36]: I love the use of ES6 in this component.

**Review by bkwon94 — COMMENTED** [review]
Good work! Clean code with good naming practices. I like how you use object destructuring, ternary operators etc. It definitely makes the code easier to read.

File: client/src/components/Carousel.jsx (hunk 1 above)
Comment (brockross) [reply client/src/components/Carousel.jsx:17]: This particular use of 'flex' actually controls the grow/shrink ratio of a given item relative to the other items in the container, but yes! `display: flex` and `flex-direction: row` do control the row-styled appearance of all the items in my carousel. Flexbox rules for one-dimensional rendering like this.

File: client/src/components/Carousel.jsx (hunk 2 above)
Comment (brockross) [reply client/src/components/Carousel.jsx:21]: Thanks! Just learning them myself for this component

**Pull request merged**